style.css

```diff
@@ -0,0 +1,43 @@
+/* reset */
+* {
+  padding: 0;
+  margin: 0;
+  box-sizing: border-box;
+}
+html {
+  height: 100%;
+  width: 100%
+}
+body {
+  width:100%;
+  height: 100%;
+  margin: auto;
+  font-family: Verdana, sans-serif;
+  background: linear-gradient(-90deg, #C07946, #995028);
+    text-align: center;
+}
+li {
+  font-family: Verdana, sans-serif;
+  list-style: none;
+}
+h3 {
+  font-family: Georgia, serif;
+  margin-top: 20px;
+}
+h1 {
+  font-family: Georgia, serif;
+  text-align: center;
+  display: inline-block;
+  font-size: 4em;
+}
+img {
+  width: 800px;
+  margin-left: 50px;
```

Comment:
You can center the fish using margin: 0 auto (or some other number for top/bottom margin)